### PR TITLE
Increase zoom

### PR DESCRIPTION
**Pull request by Ariana-B**
Increased the max zoom level of the interactive map to better see the thumbnail image, and put back css styling that was accidentally removed in a previous PR.

----
:books: Documentation preview :books:: https://datacube-explorer--501.org.readthedocs.build/en/501/

### CODE REVIEW

**Review by robbibt — APPROVED** [review]
This looks fantastic, thanks @Ariana-B!

File: cubedash/static/base.css [hunk 1]
```diff
@@ -1158,6 +1158,11 @@ details > summary {
   max-height: none !important;
 }
 
+/* Unset image overlay width to preserve correct image aspect ratio */
+.leaflet-container img.leaflet-image-layer {
```
Comment (pindge) [inline cubedash/static/base.css]: Please put in the change to `.scss` file

**Review by pindge — CHANGES_REQUESTED** [review]

File: cubedash/static/base.css (hunk 1 above)
Comment (pindge) [reply cubedash/static/base.css]: instruction is here https://github.com/opendatacube/datacube-explorer#how-do-i-modify-the-cssjavascript

Comment (Ariana-B) [reply cubedash/static/base.css]: What `.scss` file are you referring to?

Comment (pindge) [reply cubedash/static/base.css]: this one https://github.com/opendatacube/datacube-explorer/blob/develop/cubedash/static/base.sass

Comment (Ariana-B) [reply cubedash/static/base.css:1162]: When I run `make style` it doesn't do anything

Comment (Ariana-B) [reply cubedash/static/base.css]: Done

**Review by pindge — APPROVED** [review]

**Review by omad — APPROVED** [review]

**Pull request merged**